Mlem/Icons.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 import SwiftUI
 
 /// SFSymbol names for icons
-struct Icons {
+enum Icons {
```

Comment:
I'm not sure why this changed, I think xcode did it automatically? Can revert if needed

Reply:
Hmm, possibly. I don't think it makes a difference code-wise, but I'd rather keep it as a struct